rules/@typescript-no-watch/@typescript-no-watch.md

```diff
@@ -0,0 +1,108 @@
+# effector/@typescript-no-watch
+
```

Comment:
In my mind, we should warn a user about issues with JS-way in this rule right here.